integrators/cohen_hubbard_oesterwinter.hpp

```diff
@@ -0,0 +1,34 @@
+﻿
+#pragma once
+
+#include "numerics/fixed_arrays.hpp"
+
+namespace principia {
+namespace integrators {
+namespace internal_cohen_hubbard_oesterwinter {
+
+using numerics::FixedVector;
+
+// Definition of a Cohen-Hubbard-Oesterwinter formula.  |order_| is the order of
+// the approximation, that is, the error on the derivative is h^(order + 1).
+// The formula requires |order_| values of the acceleration.  numerators[0]
+// corresponds to f"(x₀), numerators[1] to f"(x₋₁), etc.
+// TODO(phl): This struct shows up in many places.  Factor it out.
```

Comment:
I don’t understand this TODO.

Reply:
We have 3 or 4 occurrences of a struct containing a `FixedVector` for numerators and a single value for the denominator.